<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="FirstLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>393</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>First集展示</string>
    </property>
   </widget>
   <widget class="QLabel" name="FollowLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Follow集展示</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GrammerText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>203</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="GrammerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>173</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>扩充后输入文法展示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FirstBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>386</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点击展示First集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FollowBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>633</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>点击展示Follow集</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SLRText">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="SLRLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>183</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SLR1文法判断</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SLRBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>166</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SLR1判断</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LR0DfaBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>163</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LR0_DFA表展示</string>
    </property>
   </widget>
   <widget class="QLabel" name="LR0Label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>183</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LR0_DFA图分析结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LR1DfaBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>393</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LR1_DFA表展示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LR1TabelBtn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>633</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LR1_分析表展示</string>
    </property>
   </widget>
   <widget class="QLabel" name="LR1DfaLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>403</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LR1_DFA图分析结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="LR1TabelLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>643</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LR1分析表结果</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="InputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户的文法输入界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AnalysisBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开始分析</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="FirstText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>321</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="FollowText">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>671</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="LR0Text">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="LR1DfaText">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>431</y>
      <width>801</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="LR1ToTabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>671</y>
      <width>801</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="version1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LR1文法分析器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>作者：20222121007黄泽基</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开源：github.com/vajackye/SCNU-CS2022-Courses</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1167</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="separator"/>
    <addaction name="UsingText"/>
    <addaction name="Quit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="UsingText">
   <property name="text">
    <string>使用说明书</string>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
